standard/basic-concepts.md

```diff
@@ -935,6 +953,7 @@ The behavior of the garbage collector can be controlled, to some degree, via sta
 
 > *Example*: Since the garbage collector is allowed wide latitude in deciding when to collect objects and run finalizers, a conforming implementation might produce output that differs from that shown by the following code. The program
 >
+> <!-- Example: {template:"standalone-console",name:"MemoryManagement1", expectedOutput:["??ordering"]} -->
```

Comment:
I'd expected to see this as UndefinedExample - isn't the idea that you're adding examples with the metadata described in #646, so we can easily identify what can be fixed with tooling support for each type of problem?